### PR TITLE
Disable actionsColumn for FC grids

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Disable actionsColumn for FC grids.
fc_1

fc_2

fc_3

Also reduced the width of the ID column on the "Regions" tab (200 wider than necessary).

### Why is it needed?
Clicking the gear icon has no effect.

### Related issue(s)/PR(s)
None

### CODE REVIEW

File: manager/assets/modext/widgets/fc/modx.panel.fcset.js [hunk 1]
```diff
@@ -298,14 +299,15 @@ MODx.grid.FCSetTabs = function(config) {
...
+        ,showActionsColumn: false
         ,fields: ['id','action','name','form','other','rank','visible','label','type']
         ,autoHeight: true
         ,plugins: [this.vcb]
         ,stateful: false
         ,columns: [{
             header: _('tab_id')
             ,dataIndex: 'name'
-            ,width: 200
+            ,width: 100
```
Comment (JoshuaLuckers) [inline manager/assets/modext/widgets/fc/modx.panel.fcset.js:310]: This should be in a separate commit.

**Comment by Ruslan-Aleev** [comment]
Corrected commits.

**Review by JoshuaLuckers — APPROVED** [review]
